Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/SettingsFragmentPresenter.java

```diff
@@ -253,6 +253,7 @@ private void addGraphicsSettings(ArrayList<SettingsItem> sl)
 		}
 		else
 		{
+			mSettings.get(SettingsFile.SETTINGS_DOLPHIN).put(SettingsFile.SECTION_CORE, new SettingSection(SettingsFile.SECTION_CORE));
```

Comment:
It's actually a negative if statement (note the `!`) so this is the correct spot. I honestly don't know why this statement is a negative if there's an else? I got confused by it too, so I can flip it around if you want.